Root/OptimizationDump.cxx

```diff
@@ -35,7 +35,11 @@
 namespace HF = HelperFunctions;
 namespace VD = VariableDefinitions;
 
-#define ARRAY_INIT {-99, -99, -99, -99}
+
+#define N_LARGE_R 8
+
+#define ARRAY_INIT {-99, -99, -99, -99, -99, -99, -99, -99}
+#define ARRAY_INIT_2 {0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
Dumb, but I want to init values to 0 instead of -99. Perhaps not necessary now that I re-init to 0 always lower.